tests/build_language_container.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+pushd  $SCRIPT_DIR &> /dev/null
+poetry build
+
+
+RELEASE_BUILD_STEP_DIST_DIRECTORY="./../language_container/exasol-bucketfs-utils-python-container/flavor_base/release/dist"
```

Comment:
```suggestion
RELEASE_BUILD_STEP_DIST_DIRECTORY="$SCRIPT_DIR/../language_container/exasol-bucketfs-utils-python-container/flavor_base/release/dist"
```

Thus, it's independent of the from where you call the script